### PR TITLE
Update ba2016layer_1.md

### PR DESCRIPTION
**Pull request by ayushchauhan**

### CODE REVIEW

File: _summaries/ba2016layer_1.md [hunk 1]
```diff
@@ -1,19 +1,66 @@
...
+
+$$ \mu^l = \frac{1}{H} \sum_{i=1}^{H}a_i^l \qquad \sigma^l = \sqrt{\frac{1}{H}\sum_{i=1}^{H} \left ( a_i^l-\mu_i^l\right )^2}$$
+
+Note that the mean and variance above are defined for a layer and are same for all neurons in that layer.
+
+### Why is layer normalization superior to batch normalization?
+
+* As shown above, batch normalization computes the normalization parameters over a mini-batch of samples which makes it sensitive to the batch size. Therefore, one cannot reduce the batch size arbitrarily. Layer normalization, on the other hand, does not put a constraint on the batch size during training and hence, can be applied even with batch size = 1 as in online learning.
+* For reasons similar to that in previous point, it is not easy to apply batch normalization in RNNs because one would need to compute and store the parameters for **each time-step across sequences** in a batch and that is a problem when the sequence length varies a lot as it commonly does. Layer normalization does not run into this problem because it computes the statistics from the inputs to all nodes in layer at the **current time-step**.
```
Comment (biofizzatreya) [inline _summaries/ba2016layer_1.md:42]: You can add a point here about the test and training steps being the same for layer normalization

File: _summaries/ba2016layer_1.md [hunk 2]
```diff
@@ -1,19 +1,66 @@
...
+
```
Comment (biofizzatreya) [inline _summaries/ba2016layer_1.md:43]: You can also talk a bit about how the normalization reduces learning rate, and stabilizes learning. Maybe add equation 14.

File: _summaries/ba2016layer_1.md (hunk 1 above)
Comment (ayushchauhan) [reply _summaries/ba2016layer_1.md:42]: I actually had that point initially but later thought it is not really something that makes batch normalization superior. Its a point in favor of batch normalization for sure. Will include it back.

File: _summaries/ba2016layer_1.md (hunk 2 above)
Comment (ayushchauhan) [reply _summaries/ba2016layer_1.md:43]: Don't you think the summary is longer than average right now? I will include the point, but skip the equation probably because I'll have to also explain it a bit if I add it.

Comment (biofizzatreya) [reply _summaries/ba2016layer_1.md:43]: Yeah sure no problem, you can skip it.
